qualang_tools/addons/calibration/README.md

```diff
@@ -0,0 +1,169 @@
+# Calibration API
+
+## Introduction
+
+This module allows to easily perform most of the standard single qubit calibrations from a single python file.
+
+The following calibrations are currently available:
+* [raw_traces](#raw-traces): acquires the raw and averaged traces from the two inputs of the OPX.
+* [time_of_flight](#time-of-flight): measures the time of flight and offsets of the two inputs of the OPX.
+* [resonator_spectroscopy](#resonator-spectroscopy): performs a resonator spectroscopy.
+* [Rabi](#rabi-measurement): performs a Rabi experiment where the pulse amplitude, duration and frequency can be scanned.
+* [T1](#t1-measurement): measures the qubit T1 by scanning the time after the application of a $\pi/2$ pulse.
+* [Ramsey](#ramsey-measurement): performs a Ramsey measurement where the idle time and pulse frequency can be scanned.
```

Comment:
it would be nice to add Ramsey with `frame_rotation_2pi()` this is for the case wehre you want to detune like 10 MHz and still see good oscillations with the previously calibrated pi pulse. Gives accurate results as well

Reply:
Do you mean add a constant phase-shift for the 2nd pi/2 pulse or get the oscillations by scanning the phase-shift of the 2nd pi/2 pulse?